packages/ts/react-crud/src/index.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line import/no-unassigned-import
+import './styles.css';
```

Comment:
Hmm, what about making `.module.css` here? We could rely on Vite to bundle separate files and import `style.css` in `index.ts`. So we will be free to use any kind of class transformation.

Reply:
As discussed, will leave as is for now.